### ISSUE TITLE
path in search gets prefxed with home

### ISSUE THREAD

**Issue by chStaiger** [UtrechtUniversity/iBridges#199]
The search path variable is always prefixed with the `session.home` if it does not start with `/`. That defeats a bit the purpose of an iRODS-server-wide search.

**Comment by qubixes** [comment]
@chStaiger I don't really understand the issue. Isn't this expected behavior?

**Comment by chStaiger** [comment]
Not really.  If you are searching for `%`, i.e. all data in the iRODS instance  you have access to, you do not want this to be changed to `session.home/%`.

**Comment by chStaiger** [comment]
Test:

ibridges v0.2.2:
```
res = search_data(session, path="%/New Folder")
{'COLL_NAME': '/nluu12p/home/research-test-christine/New Folder'}
```

Should be after fix:

```
 {'COLL_NAME': '/nluu12p/home/research-test-christine/New Folder'},
 {'COLL_NAME': '/nluu12p/home/research-test-dawa/New Folder'}]
```

**Comment by qubixes** [comment]
I can see your point from an iRODS perspective. However, I do think users (such as I) might not necessarily expect the whole iRODS server to be searched with `%`. Depending on the situation, it might be safer/faster to keep the current implementation. I also think it is more consistent with the rest of the iBridges API/CLI. If you want to search the whole iRODS server, you would have to simply use `/%`, right? That doesn't seem too difficult to me.

**Comment by qubixes** [comment]
I think the this could also be solved in a different way that might be more easy to follow to the user, which is to separate the pattern and the search collection. I think that in itself creates the largest part of the confusion.

```
search_data(session, ipath, pattern)
```
Would make a lot more sense to me, but I suppose that this is mostly dictated currently by PRC behavior.
